base/multidimensional.jl

```diff
@@ -207,11 +207,11 @@ function _unsafe_getindex(::LinearIndexing, src::AbstractArray, I::AbstractArray
     D = eachindex(dest)
     Ds = start(D)
     s = 0
-    for b in eachindex(I)
+    @inbounds for b in eachindex(I)
```

Comment:
This `@inbounds` has wider scope than the original `unsafe_getindex` calls. It would be safer to individually mark the `I[b]` and `dest[d] = src[s]` lines.